io.github.hakuneko.HakuNeko.json

```diff
@@ -0,0 +1,56 @@
+{
+	"app-id": "io.github.hakuneko.HakuNeko",
+	"base": "io.atom.electron.BaseApp",
+	"base-version": "stable",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "hakuneko",
+	"separate-locales": false,
+	"rename-desktop-file": "hakuneko-desktop.desktop",
+	"rename-icon": "hakuneko-desktop",
+	"finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+	],
+	"modules": [
+		{
+			"name": "hakuneko-desktop",
+			"buildsystem": "simple",
+			"build-commands": [
+				"ar x hakuneko-desktop_*.deb",
+				"rm -f hakuneko-desktop_*.deb",
+				"tar xf data.tar.xz",
+				"rm -f control.tar.gz data.tar.xz debian-binary",
+				"cp -r usr/* opt/* /app",
+				"chmod -R a-s,go+rX,go-w /app/lib/hakuneko-desktop/hakuneko",
+				"rm -rf /app/share/icons/hicolor/1024x1024/",
+				"install hakuneko.sh /app/bin/hakuneko",
+				"install -Dm644 io.github.hakuneko.HakuNeko.xml /app/share/appdata/io.github.hakuneko.HakuNeko.xml"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"only-arches": ["x86_64"],
```

Comment:
No need to specify it as you already have `flathub.json` specifying available arches.

Reply:
Should I just remove the sources section altogether?

Reply:
How will flatpak-builder know what to download if you remove it all? I mean just the `only-arches` line.

Reply:
Ok, I removed the only-arches line in the latest commit.